commands.c

```diff
@@ -53,6 +52,13 @@ command_answer *send_command(char *name, char *data, task_context *context)
     // we can now wake up the driver to send out a command for processing
     wake_up_interruptible(&command_wait_queue);
 
+    if (cmd->is_message)
+    {
+        return NULL;
+    }
+
+    init_waitqueue_head(&cmd->wait_queue);
```

Comment:
I think this should be initialized before line number 53. (wake_up_interruptible)